samples: applications: Use ipc_radio as default radio firmware. (#15435)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Thread samples
--------------

> The Thread samples which used one of the following radio core firmware:
> * :ref:`zephyr:nrf-ieee802154-rpmsg-sample`
> * :ref:`multiprotocol-rpmsg-sample`
> now use the :ref:`ipc_radio`.

Reviewer: ```suggestion
Thread samples that used :ref:`zephyr:nrf-ieee802154-rpmsg-sample` or :ref:`multiprotocol-rpmsg-sample` radio core firmware, now use the :ref:`ipc_radio`.
```

Zigbee samples
--------------

> The Zigbee samples which used one of the following radio core firmware:
> * :ref:`zephyr:nrf-ieee802154-rpmsg-sample`
> * :ref:`multiprotocol-rpmsg-sample`
> now use the :ref:`ipc_radio`.

Reviewer: ```suggestion
Zigbee samples that used :ref:`zephyr:nrf-ieee802154-rpmsg-sample` or :ref:`multiprotocol-rpmsg-sample` radio core firmware, now use the :ref:`ipc_radio`.
```
